### ISSUE TITLE
Can we use this in the browser to compile and run ts?

### ISSUE THREAD

**Issue by quantuminformation** [theblacksmith/typescript-compiler#18]

**Comment by svallory** [comment]
Never tried it actually, but I think it will work. It hasn't been maintained for a long time though

**Comment by IlijazM** [comment]
Yes sure but the real question is why should you do this? Let's say you only serve the typescript files and a typescript compiler, that would mean that whenever someone visits your website they first have to run the typescript compiler in order to execute javascript code. This is just inefficient. There are websites, however, that provide an online typescript sandbox (e.g. https://www.typescriptlang.org/play). This could be a use case for a typescript compiler that gets executed in a browser.
